src/components/Table/AsyncTable/MockApiClient.js

```diff
@@ -2,6 +2,7 @@
  * Simulates a client capable of asynchronously fetching
  * paginated, filtered and sorted data from some resource (e.g. an HTTP API)
  */
+/* istanbul ignore file */
```

Comment:
Instead of ignoring coverage, tests should be written

Reply:
This is a mock api file used to create data for the stories. This does not need to be included in our test coverage.